capstone-backend/src/main/java/com/google/sps/servlets/BookListServlet.java

```diff
@@ -46,7 +47,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
       ApiFuture<WriteResult> futureUsers = db.collection("booklists").document(userBookList.getID()).set(userBookList);
```

Comment:
Should this be updated to use the Firebase ID rather than the UUID as we discussed the other week?

Reply:
Done